x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -3484,7 +3484,6 @@
     "kibana-react.solutionNav.collapsibleLabel": "折叠侧边导航",
     "kibana-react.solutionNav.mobileTitleText": "{solutionName} 菜单",
     "kibana-react.solutionNav.openLabel": "打开侧边导航",
-    "kibana-react.splitPanel.adjustPanelSizeAriaLabel": "按左/右箭头键调整面板大小",
```

Comment:
Could you change the label of the translation to `console.splitPanel.adjustPanelSizeAriaLabel` instead of removing it? Thanks)